slip-0039.md

```diff
@@ -294,6 +294,12 @@ The wordlist mandated by this SLIP is [available here](slip-0039/wordlist.txt).
 
 (see the [test](slip-0039/test_wordlist.sh) which checks whether these criteria are fulfilled).
 
+## Specification for backing up [BIP-0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) Hierarchical Deterministic Wallets
+
+SLIP-0039 can be used to back up any master secret *S* which satisfies the length constraints described above. However, any application implementing SLIP-0039 for backing up a [BIP-0032](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) Hierarchical Deterministic Wallet MUST use the [BIP-0032 master seed](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#master-key-generation) as the SLIP-0039 master secret *S*. To clarify, this is the initial generated entropy of 128-512 bits, which is used as the input to `HMAC-SHA512` for deriving the BIP-0032 master node.
```

Comment:
I am afraid the last statement may cause more confusion than clarification. I would avoid any mention of the word "entropy" especially in connection with the word "initial", because "initial entropy" is exactly the term used in BIP39, so  we run the risk that someone might confuse this with the BIP39 initial entropy. BIP32 refers to this as "seed", "seed value", "seed byte sequence" or "master seed", so let's use one of these terms instead. How about "this is the initial generated seed byte sequence of 128-512 bits".